xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -209,7 +204,6 @@
       "items": {
         "type": "object",
         "properties": {
-          "playercoreid": { "type": "integer", "minimum": 1, "required": true },
           "name": { "$ref": "Global.String.NotEmpty", "required": true },
           "type": { "type": "string", "enum": [ "internal", "external", "remote" ], "required": true },
```

Comment:
Since I don't know what "pa" stands for I don't know if "pa" or "music" is a better match but "video" and "music" would be the obvious player type choices for me.
Concerning "upnp" vs. "remote" I don't have any strong opinion. "upnp" is more specific but I'm not sure if a remote could gain anything from knowing the protocol used for remote playback.